code/modules/mob/living/silicon/robot/robot_mob.dm

```diff
@@ -346,10 +346,11 @@ GLOBAL_LIST_INIT(robot_verbs_default, list(
 	var/static/list/special_modules = list(
 		"Combat" = image('icons/mob/robots.dmi', "security-radial"),
 		"Security" = image('icons/mob/robots.dmi', "security-radial"),
-		"Destroyer" = image('icons/mob/robots.dmi', "droidcombat"))
+		"Destroyer" = image('icons/mob/robots.dmi', "droidcombat"),
+		"Hunter" = image('icons/mob/robots.dmi', "xeno-radial"))
 
 	if(mmi?.alien)
-		return list("Hunter" = image('icons/mob/robots.dmi', "xeno-radial"))
```

Comment:
What about the image information being removed here?

Reply:
It's in force modules now

Reply:
Wow I am blind